KeeTrayTOTP.Tests/VersionInformationTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace KeeTrayTOTP.Tests
+{
+    [TestClass]
+    public class VersionInformationTests
+    {
+        private readonly string[] VersionInformationLines = File.ReadAllLines("version_manifest.txt");
+
+        [TestMethod]
+        public void VersionFile_ShouldHaveCorrectLength()
+        {
+            VersionInformationLines.Should().HaveCountGreaterOrEqualTo(3, "File should be a minimum of 3 lines");
+        }
+
+        [TestMethod]
+        public void VersionFile_ShouldNotContainEmptyLines()
+        {
+            VersionInformationLines.Should().NotContain("", "File should not contain empty lines");
+        }
+
+        [TestMethod]
+        public void VersionFile_ShouldStartAndEndWithSeparators()
+        {
+            var separator = VersionInformationLines.First();
+            separator.Should().HaveLength(1);
+            VersionInformationLines.Last().Should().Be(separator, "File should start and end with the same separator");
+        }
+
+        [TestMethod]
+        public void VersionFile_ShouldContainSingleEntryForPluginWithCurrentVersion()
```

Comment:
This test will fail in the future when we merge #97 because the assembly version will be different from version_manifest.txt.

Reply:
But we could filter the tests with [Running selective unit tests - .NET Core | Microsoft Docs](https://docs.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests)

Reply:
or [https://docs.microsoft.com/de-de/dotnet/core/tools/dotnet-test#filter-option-details](https://docs.microsoft.com/de-de/dotnet/core/tools/dotnet-test#filter-option-details)

Reply:
That is indeed something that we should take into account. I think the version manifest file should at all times reflect the version in `AssemblyFileVersionAttribute`. I'd like the versioning to be automated and have the unit tests act as a failsafe. Now that this PR is in master I'll merge that into #97 and see how the tests hold up.

I did not yet investigate on how to do the automatic versioning during CI, but it should bump the version in the manifest as well.